### PR TITLE
[IMP] Calendar: Calendar Usability.

### PR DESCRIPTION
**Pull request by atp-odoo**
Task: https://www.odoo.com/web?#id=19846&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.kYc2rPPGYtZfTCLk

### CODE REVIEW

File: addons/calendar/calendar_view.xml [hunk 1]
```diff
@@ -36,6 +36,26 @@
         </field>
     </record>
 
+    <record id="view_calendar_alarm_form" model="ir.ui.view">
```
Comment (atp-odoo) [inline addons/calendar/calendar_view.xml]: give id based on new guidlines. like calendar_alarm_view_form

File: addons/calendar/calendar_view.xml [hunk 2]
```diff
@@ -36,6 +36,26 @@
...
+            <field name="name">Calendar Alarm form</field>
```
Comment (atp-odoo) [inline addons/calendar/calendar_view.xml]: same here. calendar.alarm.form

File: addons/calendar/calendar.py [hunk 3]
```diff
@@ -583,6 +583,17 @@ def _get_duration(self, cr, uid, ids, field_name, arg, context=None):
         'interval': 'hours',
     }
 
+    def onchange_duration_interval(self, cr, uid, ids, duration, interval, context=None):
+        return {'value': {'name': str(duration) + ' ' + interval}}
+
+    def name_get(self, cr, uid, ids, context=None):
+        result = []
+        if not ids:
+            return result
```
Comment (atp-odoo) [inline addons/calendar/calendar.py]: no need to check this.

**Comment by dmo-odoo** [comment]
Merged at 37351983ed90f5904d4db73adadf00c0fa402e6d.

**Pull request closed without merging**